src/track/beatgrid.cpp

```diff
@@ -365,4 +385,8 @@ void BeatGrid::setBpm(double dBpm) {
     emit updated();
 }
 
+void BeatGrid::setDownbeatOffset(int offset) {
```

Comment:
In which unit is `offset` measured? Please always add Frames/Samples/Millis/Beats as a suffix.